src/components/basicComponents/BaseButton.vue

```diff
@@ -38,7 +39,7 @@ defineProps({
     type: String,
     required: true,
     validation: (variant) =>
-      ["primary", "secondary", "neutral"].includes(variant),
+      ["primary", "secondary", "neutral", "focused"].includes(variant),
```

Comment:
I wouldn't use name focus/ed as it is really close to pseudo class

Reply:
ok this is already changed in next pr :)